modules/consul-cluster/README.md

```diff
@@ -3,7 +3,7 @@
 This folder contains a [Terraform](https://www.terraform.io/) module to deploy a 
 [Consul](https://www.consul.io/) cluster in [AWS](https://aws.amazon.com/) on top of an Auto Scaling Group. This module 
 is designed to deploy an [Amazon Machine Image (AMI)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) 
-that has Consul installed via the [install-consul](https://github.com/hashicorp/terraform-aws-consul/tree/master/modules/install-consul) module in this Module.
+that has Consul installed via the [install-consul](../../modules/install-consul) module in this Module.
```

Comment:
It's irritating, but unfortunately, the Terraform Registry currently requires absolute URLs in READMEs.

Reply:
Ok I see why they were absolute now. I will change it back